packages/core/test/integration/query-interface/show-constraint.test.ts

```diff
@@ -0,0 +1,249 @@
+import { expect } from 'chai';
+import { DataTypes, Op } from '@sequelize/core';
+import { sequelize } from '../support';
+
+const queryInterface = sequelize.queryInterface;
+const dialect = sequelize.getDialect();
+
+describe('QueryInterface#showConstraints', () => {
```

Comment:
Considering this test suite is (almost) identical with addConstraint, I think we could name this suite `#{add,showConstraints}`, and delete the other one

Reply:
As above, I'll merge add, show and remove constraints into one test.